lib/infra/operators.js

```diff
@@ -41,6 +40,7 @@ class Operators {
         const destTempPath = path.join('/tmp', uuidv4());
 
         try {
+            await this.run(`mkdir -p ${path.dirname(location)}`, user, undefined, target);
```

Comment:
This will not work on windows. Use a proper fs call.

Reply:
resolved in 0e05129.

Reply:
Fixed.